specification/liftrqumulo/Qumulo.Management/main.cadl

```diff
@@ -0,0 +1,178 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-autorest";
```

Comment:
could you add an 'examples' folder in this folder, which should contain the subfolder for each api-version, e.g.
examples/
   2022-06-27-preview/
        FileSystems_CreateOrUpdate.json
         .....
   2022-10-12-preview/
        FileSystems_CreateOrUpdate.json
         .....

Reply:
@jianyexi - Provided cadl-project.yaml has issue at 

Location: [specification/liftrqumulo/Qumulo.Management/cadl-project.yaml#L1](https://github.com/Azure/azure-rest-api-specs/blob/0dc5b5f520e9af48f7487d44785df65a9b6ccdd3/specification/liftrqumulo/Qumulo.Management/cadl-project.yaml#L1)

Please suggest the right value